ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -570,6 +570,38 @@ def Conv3D(
     )
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    data_format="NHWC",
+    dilations=[1, 1, 1, 1],
+    name="Conv2D",
```

Comment:
You should include all the arguments of the original function. `use_cudnn_on_gpu` and `explicit_paddings` are missing here. I think we can't add support for `use_cudnn_on_gpu` (it still should be included though). As for `explicit_paddings`, it can be supported using ivy's conv2d function.

Reply:
This is the original documentation if I'm not mistaken https://www.tensorflow.org/api_docs/python/tf/raw_ops/Conv2D.

Reply:
ok I will do it right now.

Reply:
i will set the cudnn_on_gpu to False